client/src/index.ejs

```diff
@@ -7,15 +7,20 @@
     <link rel="shortcut icon" href="/favicon.ico">
     <script src="https://use.fontawesome.com/51ad00bb8c.js"></script>
     <script async data-cfasync='false' src='https://d29l98y0pmei9d.cloudfront.net/js/widget.min.js?k=Y2xpZW50SWQ9NzQ3Jmhvc3ROYW1lPWFrdm9sdW1lbi5zdXBwb3J0aGVyby5pbw=='></script>
-    <script>
+    <script type="text/javascript">
+      const url = new URL(window.location.href);
```

Comment:
There is no need to create a new `URL` ... `location` is already an object, that object has the `pathname` property.
* https://developer.mozilla.org/en-US/docs/Web/API/Window/location

Reply:
changed!